dist/stat_data.py

```diff
@@ -820,19 +820,26 @@ def __init__(self, name, desc, flags=''):
-    CompressStat('compress_hist_ratio_8', 'number of blocks with compress ratio smaller than 8'),
-    CompressStat('compress_hist_ratio_16', 'number of blocks with compress ratio smaller than 16'),
-    CompressStat('compress_hist_ratio_32', 'number of blocks with compress ratio smaller than 32'),
-    CompressStat('compress_hist_ratio_64', 'number of blocks with compress ratio smaller than 64'),
-    CompressStat('compress_hist_ratio_max', 'number of blocks with compress ratio greater than 64'),
     CompressStat('compress_precomp_intl_max_page_size', 'compressed page maximum internal page size prior to compression', 'no_clear,no_scale,size'),
     CompressStat('compress_precomp_leaf_max_page_size', 'compressed page maximum leaf page size prior to compression ', 'no_clear,no_scale,size'),
-    CompressStat('compress_read', 'compressed pages read'),
-    CompressStat('compress_write', 'compressed pages written'),
-    CompressStat('compress_write_fail', 'page written failed to compress'),
-    CompressStat('compress_write_too_small', 'page written was too small to compress'),
+    CompressStat('compress_read', 'compressed blocks read'),
+    CompressStat('compress_read_ratio_hist_2', 'compressed blocks read with compression ratio smaller than  2'),
+    CompressStat('compress_read_ratio_hist_4', 'compressed blocks read with compression ratio smaller than  4'),
+    CompressStat('compress_read_ratio_hist_8', 'compressed blocks read with compression ratio smaller than  8'),
+    CompressStat('compress_read_ratio_hist_16', 'compressed blocks read with compression ratio smaller than 16'),
+    CompressStat('compress_read_ratio_hist_32', 'compressed blocks read with compression ratio smaller than 32'),
+    CompressStat('compress_read_ratio_hist_64', 'compressed blocks read with compression ratio smaller than 64'),
+    CompressStat('compress_read_ratio_hist_max', 'compressed blocks read with compression ratio greater than 64'),
+    CompressStat('compress_write', 'compressed blocks written'),
+    CompressStat('compress_write_ratio_hist_2', 'compressed blocks written with compression ratio smaller than  2'),
+    CompressStat('compress_write_ratio_hist_4', 'compressed blocks written with compression ratio smaller than  4'),
+    CompressStat('compress_write_ratio_hist_8', 'compressed blocks written with compression ratio smaller than  8'),
+    CompressStat('compress_write_ratio_hist_16', 'compressed blocks written with compression ratio smaller than 16'),
+    CompressStat('compress_write_ratio_hist_32', 'compressed blocks written with compression ratio smaller than 32'),
+    CompressStat('compress_write_ratio_hist_64', 'compressed blocks written with compression ratio smaller than 64'),
+    CompressStat('compress_write_ratio_hist_max', 'compressed blocks written with compression ratio greater than 64'),
+    CompressStat('compress_write_fail', 'block written failed to compress'),
```

Comment:
Another thing to call out is in the descriptions I've changed references from "pages compressed" to "blocks compressed" since the surrounding code refers to blocks rather than pages. This makes sense for WT devs but I'm interested on input if this is helpful/harmful for other T2 users to see blocks instead of pages

Reply:
@agorrod, it seems that we are using pages and blocks interchangeably while I agree that blocks should be used here. What do you think?

Reply:
My preference is to stick with "page" generally (though I probably am guilty of being inconsistent with that).

Technically: the block manager allocates and tracks space utilization in the file. That space is tracked as blocks - where it is agnostic to the content being stored (hence it's called a block, instead of a page - which has a known structure and content).

There isn't a clear line in this case, but I think of compression as at the page level. i.e: we've laid out a page and are now compressing that content before storing it at the right place in the file via the block manager. On the other hand we do term this "block compression" in some places - hence personal preference comes in.

tl;dr:
I have a strong preference for consistency (where it doesn't involve wholesale changes of things already in the field), I have a mild preference for "page", but could be convinced that "block" is the better term here. If we go with page, maybe an additional "on-disk page" is worthwhile here, since we have so many different types of pages..

Reply:
SGTM. Most of our existing statistics - other than the compression histogram stats that we want to reword - use pages already so I'll align with existing usage. I've changed back to using pages in [5cf36f6](https://github.com/wiredtiger/wiredtiger/pull/9908/commits/5cf36f69289817a2b05ed4c43b994cc1fc16ab1b)